rclpy/rclpy/node.py

```diff
@@ -1091,6 +1091,21 @@ def remove_waitable(self, waitable: Waitable) -> None:
         self.__waitables.remove(waitable)
         self._wake_executor()
 
+    def resolve_topic_or_service_name(
```

Comment:
At this layer, it seems a little clunky to me to pass a separate flag for a service. Though I don't have particular use-case in mind, I think I would prefer two separate functions:

- `resolve_topic_name()`
- `resolve_service_name()`

Reply:
Comment taken